src/functions/avl-subscriptions/index.ts

```diff
@@ -17,7 +17,7 @@ import { ZodError, z } from "zod";
 const requestParamsSchema = z.preprocess(
     Object,
     z.object({
-        subscriptionId: z.union([createStringLengthValidation("subscriptionId"), z.literal("")]),
+        subscriptionId: z.union([createStringLengthValidation("subscriptionId"), z.literal("")]).optional(),
```

Comment:
```suggestion
        subscriptionId: z.union([createStringLengthValidation("subscriptionId"), z.literal("")]).nullable(),
```
would nullable be better here to also allow null values?